src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModalContainer.tsx

```diff
@@ -20,8 +20,8 @@ interface ActivateWorkPackageModalContainerProps {
 }
 
 export interface FormInput {
-  projectLeadId: number | '';
-  projectManagerId: number | '';
+  projectLeadId?: number;
```

Comment:
idk if you want me to just put this back, the fact that were using Form Input for default values maybe doesnt make much sense anymore? but i can also make the default values be like -1 if we dont want it to be optional?

Reply:
ehhh it's fine for now, this whole solution for autocompletes is not my favorite, but tbh it works, and you made it look clean with the toasts, so I'm cool with it